packages/repository/test/unit/repositories/relation-factory.unit.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  hasManyRepositoryFactory,
+  juggler,
+  DefaultCrudRepository,
+  Entity,
+  HasManyDefinition,
+  RelationType,
+  ModelDefinition,
+} from '../../../src';
+import * as RelationFactoryNamespace from '../../../src/repositories/relation.repository'; // needed to spy on the factory function
+import {sinon} from '@loopback/testlab';
+
+describe('hasManyRepositoryFactory', () => {
+  let meta: HasManyDefinition;
+  let targetRepo: DefaultCrudRepository<Entity, number>;
+  let db: juggler.DataSource;
+
+  before(givenDataSource);
+  before(givenOrderRepository);
+
+  context('DefaultHasManyEntityCrudRepository', () => {
+    it('', () => {
+      meta = {
+        type: RelationType.hasMany,
+        modelFrom: class FromModel extends Entity {},
+        keyTo: 'customerId',
+        keyFrom: 'id',
+      };
+
+      const spy = sinon.spy(
```

Comment:
The code is in a rough state right now, but I wanted to get feedback on whether this single test case is sufficient for the unit test.